spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Location.java

```diff
@@ -192,7 +204,13 @@ JSONObject toJSONObject() {
         }
 
         JSONObject toJSONObject() {
-            return new JSONObject().put("startLine", startLine).put("endLine", endLine);
+            JSONObject json = new JSONObject().put("startLine", startLine);
+
+            if (endLine != 0) {
```

Comment:
based on the change in the constructor, I'm just checking if endline is different from zero. If yes, we have to concat the endline to the json